.github/workflows/nativeBuild.yml

```diff
@@ -142,9 +142,11 @@ jobs:
         include:
           - os: ubuntu-latest
             platform: linux
+            container: alpine:3.17
```

Comment:
as per https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container